### PR TITLE
added `g daily rec N`

### LINKED ISSUE #72: support graphs for `daily rec N`
- [ ] Green color please
- [ ] Support N parameter too

### PR DESCRIPTION
**Pull request by baljinderxd**
closes #72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,9 @@
 | daily active N      |       Daily active cases analysis for past `N` days       | :heavy_check_mark: |   `daily active 15` |
 | g daily active N    |   Daily active cases graphical analysis for past `N` days | :heavy_check_mark: | `g daily active 15` |
 | daily rec           |      Daily recovered cases analysis for past 10 days      | :heavy_check_mark: |         `daily rec` |
+| g daily rec         | Daily recovered cases graphical analysis for past 10 days | :heavy_check_mark: |       `g daily rec` |
 | daily rec N         |      Daily recovered cases analysis for past `N` days     | :heavy_check_mark: |      `daily rec 15` |
+| g daily rec N       | Daily recovered cases graphical analysis for past 10 days | :heavy_check_mark: |    `g daily rec 15` |
```
Comment (gurrrung) [inline README.md:22]: *N days

File: app/commands/dailyRecGraph.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const prepareAnswer = require('../utils/prepareAnswer');
```
Comment (gurrrung) [inline app/commands/dailyRecGraph.js:1]: Let's follow the convention and keep the name of this file as `dailyRecoveredGraph.js`

**Review by gurrrung — CHANGES_REQUESTED** [review]
Need minor changes and then we are ready to merge this.

File: README.md (hunk 1 above)
Comment (baljinderxd) [reply README.md:22]: Oh sorry, correcting right now😬

File: app/commands/dailyRecGraph.js (hunk 2 above)
Comment (baljinderxd) [reply app/commands/dailyRecGraph.js:1]: okay no problem changing it 👍

**Review by gurrrung — APPROVED** [review]
LGTM

**Pull request merged**